client/packages/cm-tarnation/src/buffer.ts

```diff
@@ -0,0 +1,333 @@
+import { search, SearchOpts } from 'wj-util'
```

Comment:
Shouldn't eslint be adding terminating semicolons to these files? Like we discussed in chat, developers use whatever local style, then eslint normalizes it.

Reply:
I'm leaving ESLint rules alone until we all get together to discuss it.